connectors/src/lib/edit_distance.ts

```diff
@@ -0,0 +1,41 @@
+// Returns the Levenshtein distance between str1 and str2.
```

Comment:
why not use eg https://github.com/gustf/js-levenshtein ?

Reply:
not a big fan of having this code in this repo, since It doesn't look great due to all these "expect error"